### PR TITLE
add synctex support

### PR DESCRIPTION
**Pull request by chantisnake**
tested on Sumatra pdf.

### CODE REVIEW

File: doc/tags [hunk 1]
```diff
@@ -0,0 +1 @@
+vim-latex-live-preview.txt	vim-latex-live-preview.txt	/*vim-latex-live-preview.txt*
```
Comment (xuhdev) [inline doc/tags]: Everything is good, except that this line shouldn't be tracked.

**Comment by chantisnake** [comment]
I am sorry, that was a mistake...

**Comment by chantisnake** [comment]
I will add a variable later to control synctex, and pull again.

**Comment by chantisnake** [comment]
I have found a problem, synctex will automatically go to a tex file in temp... any idea on this?

**Comment by chantisnake** [comment]
I have figured out, you need to change the inverse search function,

I have figured out a way on windows, I think on *nix system will be pretty much the same idea, not too hard

**Comment by xuhdev** [comment]
Can you add your changes to the PR? We can discuss how to make it work on *NIX.

**Comment by chantisnake** [comment]
basically my solution is to write a Powershell script, and invoke that script when people do inverse search:
here is the script:
```powershell
function Invoke-Sumatra-Inverse-Search {
    param($tempFileFullName, $lineNumber=0)
    $filename = $tempFileFullName.split('\')[-1]  # get the file name
    $fileFullName = "$latexPath\$filename"  # latex-path is the latex work space
    gvim --remote-silent +$lineNumber "$fileFullName" # the line to invoke inverse search
}
```
here is the inverse search command:
```
PowerShell Invoke-Sumatra-Inverse-Search %f %l
```

but this spends tons of time invoking powershell, and based on the fact that I know where my latex dir is...
I think the way to do it is to create a file that contain the original file location and puts that in the temp dir, and write a python script(or cmd, sh script) that calls inverse search...

**Comment by xuhdev** [comment]
@chantisnake I think this plugin should only be responsible to generate the synctex automatically. To make use of it, you should probably query for a plugin which improves the editing aspects of LaTeX files, such as Vim-LaTeX.

**Comment by ranguli** [comment]
Is this ready to be closed?

**Pull request closed without merging**